lib/ssl/src/ssl.erl

```diff
@@ -2503,10 +2496,10 @@ assert_server_only(client, Bool, Option) ->
     role_error(Bool, server_only, Option);
 assert_server_only(_, _, _) ->
     ok.
-assert_client_only(server, Bool, Option) ->
-    role_error(Bool, client_only, Option);
-assert_client_only(_, _, _) ->
-    ok.
+%% assert_client_only(server, Bool, Option) ->
+%%     role_error(Bool, client_only, Option);
+%% assert_client_only(_, _, _) ->
+%%     ok.
```

Comment:
Remove

Reply:
fixed in [05288e3](https://github.com/erlang/otp/pull/7543/commits/05288e3f32668ad581571e0a1c3b9e0d5943ff09)